subsystem-drive/test/steer-modes_test.cpp

```diff
@@ -171,12 +171,12 @@ namespace sjsu::drive
             commands.speed = speed;
             tri_wheel_router_arguments arguments;
             arguments = SteerModes::SpinSteering(commands);
-            CHECK_EQ(arguments.back.steer.angle, SteerModes::kBackRightSpinAngle);
-            CHECK_EQ(arguments.left.steer.angle, -SteerModes::kLeftSpinAngle);
-            CHECK_EQ(arguments.right.steer.angle, -SteerModes::kBackRightSpinAngle);
+            CHECK_EQ(arguments.back.steer.angle, SteerModes::kRightLegDriveOffset);
+            CHECK_EQ(arguments.left.steer.angle, SteerModes::kLeftLegDriveOffset + 200);
```

Comment:
where does the 200 come from?

Reply:
The left.steer.angle produces a value of 200 so this caused the check to fail.